azavea/district-builder-dtl-pa
# Add plan submission management command

## Overview

Adds a management command that generates a static HTML page that provides an overview of a submission to the contest.

### Checklist


### Demo

![screenshot-2018-6-8 draw the lines pa](https://user-images.githubusercontent.com/447977/41162181-2519d836-6b03-11e8-8f71-6a4076fc822a.png)


### Notes
- The layout and content aren't quite nailed down yet; @tgilcrest is going to follow up with C70 and create another task for additional work based on their feedback. The most obvious manifestation of this is the explanation links, which don't go anywhere, but the styling is also super basic for the same reason.

## Testing Instructions

 * Run migrations `docker-compose exec django bash` and then `./manage.py migrate`
 * Turn on `./scripts/server`
 * Use the UI to submit one of your plans
 * Use the Django shell to get the pk of the plan you just submitted. If you've never submitted a plan before then it's probably 1 and you can skip this step. If you have submitted a plan then it'll be the highest-numbered PlanSubmission pk.
 * In your bash session from before, run `./manage.py submission_report <pk>`
 * Outside the VM, run `vagrant rsync-back`.
 * From `django/publicmapping` run `python -m SimpleHTTPServer` and then navigate to `http://localhost:8000`. Confirm that the output matches the screenshot above.

Closes #157604447
